plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/statistics/DeltaLakeJsonFileStatistics.java

```diff
@@ -112,22 +112,31 @@ public Optional<Map<String, Object>> getNullCount()
     @Override
     public Optional<Object> getMaxColumnValue(DeltaLakeColumnHandle columnHandle)
     {
-        Optional<Object> value = getStat(columnHandle.getPhysicalName(), maxValues);
+        if (!columnHandle.isBaseColumn()) {
+            return Optional.empty();
+        }
+        Optional<Object> value = getStat(columnHandle.getBasePhysicalColumnName(), maxValues);
         return value.flatMap(o -> deserializeStatisticsValue(columnHandle, String.valueOf(o)));
     }
 
     @Override
     public Optional<Object> getMinColumnValue(DeltaLakeColumnHandle columnHandle)
     {
-        Optional<Object> value = getStat(columnHandle.getPhysicalName(), minValues);
+        if (!columnHandle.isBaseColumn()) {
+            return Optional.empty();
+        }
+        Optional<Object> value = getStat(columnHandle.getBasePhysicalColumnName(), minValues);
         return value.flatMap(o -> deserializeStatisticsValue(columnHandle, String.valueOf(o)));
     }
 
     private Optional<Object> deserializeStatisticsValue(DeltaLakeColumnHandle columnHandle, String statValue)
     {
+        if (!columnHandle.isBaseColumn()) {
```

Comment:
nit: FYI This seems to be done as well in `deserializeColumnValue` one line below.
No change needed.

Reply:
`deserializeColumnValue` throw a verification exception to verify the column is base column.

verify(column.isBaseColumn(), "Unexpected dereference: %s", column);